contents/blog/trademarks-blog-post.md

```diff
@@ -0,0 +1,34 @@
+---
+date: 2021-08-18
+title: Don’t Bother Securing Your Trademarks In The Beginning
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: mo-shehu
+featuredImage: ../images/blog/filename.png
+featuredImageType: full
+---
+Don’t bother securing your trademarks in the early days of your business. 
+
+In fact, if you’re yet to raise a pre-seed round, the money you’d spend on IP lawyers could probably be better spent elsewhere. In those first crucial months, you’ve got bigger problems to worry about - like finding product-market fit, hiring more (and better) people, improving your quality control, ramping up your marketing and [content](https://posthog.com/blog/running-content-at-posthog), and more. There’s also a good chance that your brand might evolve during that time - from name changes and logo revamps to brand fonts and colors. This makes trademarking a word mark or logo mark _that_ early a risky bet.
+
+## What is the different between a word mark and a logo mark?
+
+A word mark is typically just the name of the brand (as written in a specific font, if applicable). In our case, it’s the word ‘PostHog.’ A logo mark is typically pictorial - in our case, the hedgehog logo either alone or combined with a stylized word (see below). The word mark is the most versatile, so that should be your first trademark priority.
+
+[insert side by side comparison between the PH word mark and logo mark here]
+
+When you decide to register your trademark, expect to pay around $3,000 on securing a trademark in the four largest territories you’ll probably do business in: the US, UK, EU, and China. That last one is especially salient, seeing as they subscribe to a “First to File” rule: the first person to file the trademark owns the full rights to use and license it. This means that even if you’ve been using the brand name ‘Samsung’ for years but never registered it in China, and someone else registers it before you, they become the legal owners of that trademark (in China). 
+
+We’ve currently secured our brand name (‘PostHog’) and word mark in the four territories listed above. The process differs slightly for each region:
+1. *US:* Register it yourself via the [US Patents and Trademarks Office website](https://www.uspto.gov/trademarks) (slower, cheaper), or hire a trademark attorney to do it for you (faster, pricier, and necessary if you live outside the US). If you go the latter route, get a fixed-fee service provider and avoid getting charged by the hour.
```

Comment:
I only suggested to use the trademark attorney in the interview, not filing it yourself. We use [Cognition IP](https://www.cognitionip.com/).

Reply:
Added more information to give the reader options.

Reply:
I don't think we should give the reader options given this is supposed to be giving direct actionable advice. In this case, my recommendation is to go with a fixed-fee agency. 

Can we please stick to what I said originally, and link to Cognition IP? That makes this advice much more actionable than 'find a trademark attorney', which is useless.